coremltools/converters/mil/debugging_utils.py

```diff
@@ -143,6 +143,7 @@ def replace_inputs(func, input_vars):
         raise ValueError(f"outputs {outputs_not_found} not found in the function.")
 
     func.set_outputs(new_outputs)
+    func.set_output_types([ct.TensorType(dtype=o.dtype) for o in new_outputs])
```

Comment:
Thank you @smpanaro !
I think this is a correct fix.
Would you mind adding one unittest to cover this PR?